app/src/controllers/groupOperations/reducer.js

```diff
@@ -71,8 +71,9 @@ export const validationErrorsReducer = (namespace) => (
       return { ...payload };
     case REMOVE_VALIDATION_ERRORS: {
       const errorIds = Object.keys(state);
+      const payloadIds = payload.toString().split(',');
```

Comment:
`payload` should be already an array. See `removeValidationErrorsAction` and `removeValidationErrorAction` in `app/src/controllers/groupOperations/actionCreators.js` and their usage (`app/src/controllers/groupOperations/sagas.js`)